src/3d/symbols/qgspoint3dsymbol.cpp

```diff
@@ -48,6 +49,11 @@ void QgsPoint3DSymbol::writeXml( QDomElement &elem, const QgsReadWriteContext &c
   QDomElement elemTransform = doc.createElement( QStringLiteral( "transform" ) );
   elemTransform.setAttribute( QStringLiteral( "matrix" ), Qgs3DUtils::matrix4x4toString( mTransform ) );
   elem.appendChild( elemTransform );
+
+  if ( billboardSymbol() )
+  {
+    elem.setAttribute( QStringLiteral( "billboard-symbol" ), QgsSymbolLayerUtils::symbolProperties( billboardSymbol() ) );
```

Comment:
this code should use QgsSymbolLayerUtils::saveSymbol() - and the current read/write context should be passed to it so that any paths can be converted to relative.